### ISSUE TITLE
Support for Origin 4.1 

### ISSUE THREAD

**Issue by gustavonalle** [gustavonalle/oc-cluster-up#9]
Recently the dind support in master has received some love in the form a PR: https://github.com/openshift/origin/pull/22293/
This should be enough to update the image to the latest code

**Comment by AkihiroSuda** [comment]
Is this still planned?

**Comment by AkihiroSuda** [comment]
dind seems removed in 4.2 :(
https://github.com/openshift/origin/pull/23177

**Comment by gustavonalle** [comment]
Yes, it stopped being maintained and now it's officially gone. The Openshift team is not keen on running it inside docker, the closest available right now is using libvirt from https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md, you may give a try

**Comment by gustavonalle** [comment]
By the way, this issue is still planned, it just won't use the dind-cluster from openshift :)

**Comment by AkihiroSuda** [comment]
Does `openshift/installer` technically/legally require non-okd OpenShift license?

**Comment by gustavonalle** [comment]
The installer itself is Apache V2, I recommend checking with ```openshift/installer``` team which exact version and/or licenses apply to the cluster it install.

**Comment by morningspace** [comment]
@gustavonalle I was looking for a `dind`-based solution for OpenShift multiple-node cluster launch to embed into my [lab-k8s-playground](https://github.com/morningspace/lab-k8s-playground)... then come across here...

Really love this project... but unfortunately, it appears dind will not be officially supported by OKD...

With that, any idea/suggestion, whether there's alternatives, and will this project still be maintained and go forward?

**Comment by gustavonalle** [comment]
I recommend you to check Code Ready from Red Hat

https://developers.redhat.com/blog/2019/09/05/red-hat-openshift-4-on-your-laptop-introducing-red-hat-codeready-containers/

It's not dind, but I had some good reviews of people using it

**Comment by morningspace** [comment]
Yeah, actually I am looking at `crc`, which looks to be the official approach recommended by Red Hat. But it seems to be single node rather than multiple nodes. Also, it's only applicable to 4.x, not 3.x.

BTW: Thanks for your prompt reply :-) @gustavonalle

**Comment by SemanticBeeng** [comment]
> By the way, this issue is still planned, it just won't use the dind-cluster from OKD :)

@gustavonalle I love this approach because it dramatically increases the adoption of `openshift` for development without forcing the Red Hat OS. 

Am a stakeholder in having it brought up to date,
Please advise on your latest take on this design : any limitations or roadblocks?
